### ISSUE TITLE
Add SoundWaves client

### ISSUE THREAD

**Issue by benzimmer** [podlove/podlove-subscribe-button#113]
Name of developer: Arvid P. L. Böttiger 
Contact information of developer: bottiger@gmail.com
Name of app or cloud service: SoundWaves
Operating System (for apps): Android
Syntax of app-specific URI for podcast subscription:
List of supported common URI schemes:
High-quality image file of the icon for the app/service:

**Comment by benzimmer** [comment]
@bottiger could you please let me know which subscription schemes the app support and provide us with a high-res logo for it?

Please also let me know when subscription scheme support hits the Play store. Thanks!

**Comment by bottiger** [comment]
@benzimmer Here is a link to the logo: https://github.com/bottiger/SoundWaves/blob/master/store/icon/launcher.png

I have implemented the schemes for adding a feed to the app:

pcast:
pcast://
itpc:
itpc://
soundwaves://subscribe

that said - I am not quite sure how to handle feeds which doesn't include http:// in the URL properly. At the moment I just assume they are http feeds, and not https. I really want to avoid doing the testing and validation of the URL inside the app.

I will let you know when the app is live. My next iteration is kind of ready for beta testing, but I'm taking a week of vacation next week.

**Comment by bottiger** [comment]
@benzimmer I completely forgot about this issue. However, the subscription schemes are implemented and live on the play store.

**Comment by timpritlove** [comment]
>that said - I am not quite sure how to handle feeds which doesn't include http:// in the URL properly. At the moment I just assume they are http feeds, and not https. I really want to avoid doing the testing and validation of the URL inside the app.

Assuming http is right but be ready to accept fully qualified https URLs as these are are now a thing and will be pretty commonplace once iTunes removes its weird limit in that regard.

**Comment by timpritlove** [comment]
Google Play store URL: <https://play.google.com/store/apps/details?id=org.bottiger.podcast>

**Comment by benzimmer** [comment]
I think using http:// as the default is just fine, because best practice is to have a http->https redirect anyway. So, as long as the app can handle that, everything should be fine.

**Comment by timpritlove** [comment]
>I think using http:// as the default is just fine, because best practice is to have a http->https redirect anyway. So, as long as the app can handle that, everything should be fine.

I disagree. Our spec at <http://podlove.org/podlove-subscribe-button/> is pretty straight forward in its demand to be able to accept https feed URLs. They exist and they will be passed on.

**Comment by benzimmer** [comment]
I meant when no protocol is given.

**Comment by timpritlove** [comment]
Ah okay. Well then it's pretty clear that "no protocol" means "http"
